pkg/cluster/nodes/util.go

```diff
@@ -44,3 +46,23 @@ func GetControlPlaneEndpoint(allNodes []Node) (string, error) {
 
 	return fmt.Sprintf("%s:%d", loadBalancerIP, loadbalancer.ControlPlanePort), nil
 }
+
+// to avoid the DNS crash we need copy host's /etc/resolv.conf to node
+// ref: https://github.com/kubernetes-sigs/kind/pull/484#issuecomment-489469044
+func addResolve(node *Node) error {
+	resolv, err := ioutil.ReadFile("/etc/resolv.conf")
```

Comment:
windows has no /etc/resolv.conf.

Reply:
this is a blocker.
/hold

Reply:
> windows has no /etc/resolv.conf.

Then I will judge through the OS first, skip the behavior of windows. I can't accurately judge the behavior of windows.

Reply:
just skip Windows. :speak_no_evil:

Reply:
this is the first time kind does not behave the same in all environments, not really a fan of that.

also, I don't think we should be reading the host's resolv.conf at all